app/grandchallenge/cases/views.py

```diff
@@ -146,3 +147,23 @@ class RawImageFileViewSet(
     queryset = RawImageFile.objects.all()
     permission_classes = [DjangoObjectOnlyWithCustomPostPermissions]
     filter_backends = [ObjectPermissionsFilter]
+
+    def get_permissions(self):
+        if self.action == "download":
+            return [IsAdminUser()]
+        return super().get_permissions()
+
+    @action(detail=True, methods=["get"])
+    def download(self, request, pk=None):
```

Comment:
Do we really need to expose this? Isn't having it in the admin panel enough?

Reply:
This code can go now, right?

Reply:
Yes, I thought I had removed it.